wildfly-subsystem-archetype/src/main/resources/archetype-resources/__artifactId__-feature-pack/src/main/resources/layers/standalone/__artifactId__/layer-spec.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<layer-spec xmlns="urn:jboss:galleon:layer-spec:2.0" name="${artifactId}">
+    <props>
+        <prop name="org.wildfly.description" value="This layer can be used to provide the ${artifactId}."/>
+    </props>
```

Comment:
Perhaps a comment with a link to the set of Glow rules that could be useful for discovery: https://docs.wildfly.org/wildfly-galleon-feature-packs/

And how to have your feature-pack integrated in the set of feature-packs known by WildFly Glow: https://github.com/wildfly/wildfly-galleon-feature-packs